test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -63,21 +62,27 @@ describe("Bitcoind", () => {
   })
 
   it("funds lnd1 node", async () => {
-    await createMandatoryUsers()
-
     const amount = 1
     const { chain_balance: initialBalance } = await getChainBalance({ lnd: lnd1 })
     const sats = initialBalance + btc2sat(amount)
 
     const funderWalletId = await getFunderWalletId()
-    const address = await Wallets.createOnChainAddress(funderWalletId)
-    if (address instanceof Error) throw address
-
-    await sendToAddressAndConfirm({ walletClient: bitcoindOutside, address, amount })
-    await waitUntilBlockHeight({ lnd: lnd1 })
+    await fundWalletIdFromOnchain({
+      walletId: funderWalletId,
+      amountInBitcoin: amount,
+      lnd: lnd1,
+    })
 
     const { chain_balance: balance } = await getChainBalance({ lnd: lnd1 })
     expect(balance).toBe(sats)
+
     await checkIsBalanced()
+
+    const balanceFunderWalletId = await getBalanceForWalletId(funderWalletId)
+    balanceFunderWalletId
+    // console.log({ balanceFunderWalletId }, "funderWalletId")
```

Comment:
From discussion at code review we decided not to keep commented code and to put comments ABOVE the line they refer to.